wgengine/magicsock/magicsock.go

```diff
@@ -2607,6 +2613,7 @@ func (c *Conn) rebind(curPortFate currentPortFate) error {
 // It should be followed by a call to ReSTUN.
 func (c *Conn) Rebind() {
 	metricRebindCalls.Add(1)
+	c.health.SetHealthy(sendToBlockedWithEPERMWarning)
```

Comment:
How long is this unhealthy then? Sub-millisecond before the code that marked it healthy then calls rebind?